### DISCUSSION TITLE
force-dynamic not working in Route Handler

### DISCUSSION THREAD

**Discussion by kjxbyz** [vercel/next.js#64450] [Help]
### Summary

next@14.2.1

```javascript
// app/backend/releases/github/list/route.ts
export const dynamic = "force-dynamic";
```
`https://www.kjxbyz.com/picguard/backend/releases/github/list/?page=1&page_size=10`

The length of `data` is 1, but there are two releases under this repo

<img width="565" alt="2024-04-14 01 15 39" src="https://github.com/vercel/next.js/assets/47768002/e2dc90c6-7ed0-42fe-b6c2-0bfd4459cb20">

**Comment by Robbilie** [comment]
it seems like I am facing the same issue…
